scripts/raport-regressions.scala

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env -S scala-cli shebang
+// #!/usr/bin/env -S scala-cli shebang
```

Comment:
We could clean this file a bit, remove commented out imports/directires and fix whitespace diffs (scalafmt should do the job)